sonar-cxx-plugin/src/test/resources/org/sonar/plugins/cxx/reports-project/coverage-reports/bullseye/coverage-result-bullseye-win.xml

```diff
@@ -1,92 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
it was converted to /bullseye/coverage-result-bullseye.xml and absolute paths taken out. the absolute paths check needs to be tested in integration tests (i think we have one test in travis doing that)